app/graphql/types/query_type.rb

```diff
@@ -0,0 +1,12 @@
+Types::QueryType = GraphQL::ObjectType.define do
+  name 'Query'
+  description 'The query root of this schema.'
+
+  field :search, !types[Types::OrganizationType] do
+    argument :term, types.String
```

Comment:
Should this be a required argument?

Reply:
given the current implementation it would need to be yes. in other search endpoints we fall back to a `match_all` query when no term is provided, but this endpoint doesn't support that.

Reply:
I made it required.